admin/src/view/index.js

```diff
@@ -122,7 +127,9 @@ const DeploymentLast = (props) => {
           </div>
           <div className="mb-4">
             <div className="label">STATE</div>
-            <div>{deployment.readyState === 'READY' ? `READY (${moment(deployment.ready).format(DATETIME_FORMAT)})` : deployment.readyState}</div>
+            <StatusWrapper state={deployment.readyState}>
```

Comment:
Date is missing here.

Reply:
I saw it same as CREATED at below this field. Why you need to duplicate that info?

Or maybe its your mistake xD I can fix it in this PR if its true.

Reply:
It was intentional. There is created dated and ready state date. You can see how long it takes to deploy. What do you think?

Reply:
I knew the purpose. But current implement Created and State was formatted from same value of `deployment.ready`.

Created should be formatted from `deployment.createdAt`.

Or if you want to show how long it takes:
`moment.duration(deployment.ready - deployment.createdAt)`

Reply:
Oh, you're right. So, please fix CREATED to use `deployment.createdAt` and in STATE keep `deployment.ready`. Thx.

Reply:
The screenshot after changed

![image](https://user-images.githubusercontent.com/14819342/103459332-3148ce00-4d41-11eb-9c3e-179baf995520.png)